docs/authentication/saml/overview.mdx

```diff
@@ -1,185 +1,81 @@
 
-**Will SAML work for my existing users?**
+### Will SAML work for my existing users?
 
-Yes, SAML will work for your existing users as well! That means, that if you enable SAML, any existing users that their email address matches the SAML Connection domain, they will have to authenticate on the IdP side and an Enterprise Account will be linked to their existing account.
+Yes, SAML will work for your existing users as well! If you enable SAML, any existing users with an email address that matches the SAML Connection domain will have to authenticate on the IdP side and an Enterprise Account will be linked to their existing account.
 
-**What happens if I have multi-factor enabled at Clerk?**
+### What happens if I have multi-factor enabled at Clerk?
 
-This will work: Once the user comes back from the IdP, they will need to go through the extra factors of authentication. This is in case you need to add extra factors on top of what your IdP supports (or in case they don’t). You can choose to not enable this feature if you wish.
+This will work: Once the user comes back from the IdP, they will need to go through the extra factors of authentication. This is in case you need to add extra factors on top of what your IdP supports (or in case they don't). You can choose to not enable this feature if you wish.
 
-**What happens if I delete the SAML connection? Will my users be deleted?**
+### What happens if I delete the SAML connection? Will my users be deleted?
 
 The users will not be deleted, so your application will not break. However, they will need to “reintroduce” themselves to your new strategies by resetting their passwords or via OTP (depending on the strategy you choose).
 
-**Does Clerk support IdP-initiated SSO?**
+### Does Clerk support IdP-initiated SSO?
 
 No, Clerk only supports SP-initiated flows at this time.
 
-**How much does it cost now?**
+### How much does it cost now?
 
-It’s going to be free during the Beta period.
+It's going to be free during the Beta period for production instances. Connections in development instances will always be free, but capped to 5.
 
-**How much will it cost after Beta?**
+### How much will it cost after Beta?
 
-It will cost $50 per active connection per month for production instances. Connections in development instances are and will be free, but capped to 5.
+It will cost $50 per active connection per month for production instances. Connections in development instances will always be free, but capped to 5.
 
-**Can I get a bulk discount?**
+### Can I get a bulk discount?
 
 Yes, [reach out to us](https://clerk.com/support) and we will work a plan out.
```

Comment:
```suggestion
 Yes, [reach out to support](https://clerk.com/support) and we will work a plan out.
```
We generally refer to Clerk as "Clerk" rather than us. This sidesteps the issue